### PR TITLE
3x tweaks

### PR DESCRIPTION
**Pull request by manolama**

### CODE REVIEW

File: storage/asynchbase/src/main/java/net/opentsdb/storage/Tsdb1xMultiGet.java [hunk 1]
```diff
@@ -448,13 +469,13 @@ void onError(final Throwable t) {
...
       state = State.EXCEPTION;
-        if (!node.push()) {
+      if (!node.push()) {
         current_result.setException(t);
         final QueryResult result = current_result;
         current_result = null;
         outstanding.set(0);
         node.onNext(result);
-      } else {
+      } else if (!node.pipelineContext().queryContext().isClosed()) {
```
Comment (sidhhu) [inline storage/asynchbase/src/main/java/net/opentsdb/storage/Tsdb1xMultiGet.java:478]: call error if query context is closed?

File: implementation/servlet/src/main/java/net/opentsdb/servlet/resources/AsyncRunner.java [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+/**
+ * Static helper to handle an async query. 
+ * 
+ * @since 3.0
+ */
+public class AsyncRunner {
+  private static final Logger LOG = LoggerFactory.getLogger(AsyncRunner.class);
+  
+  private AsyncRunner() { }
```
Comment (sidhhu) [inline implementation/servlet/src/main/java/net/opentsdb/servlet/resources/AsyncRunner.java:42]: Do you need this?

File: storage/asynchbase/src/main/java/net/opentsdb/storage/Tsdb1xMultiGet.java (hunk 1 above)
Comment (manolama) [reply storage/asynchbase/src/main/java/net/opentsdb/storage/Tsdb1xMultiGet.java:478]: Nope, cause we no longer have any place to send the error.

File: implementation/servlet/src/main/java/net/opentsdb/servlet/resources/AsyncRunner.java (hunk 2 above)
Comment (manolama) [reply implementation/servlet/src/main/java/net/opentsdb/servlet/resources/AsyncRunner.java:42]: Yeah cause I don't want anyone instantiating it. It's how you mark a static class as intending to hold static methods only..

**Pull request merged**